### PR TITLE
SPCTR - 2112 : New : Icon block - basic block

### PR DESCRIPTION
**Pull request by adi3890**
### Description
This PR consists basic block for Icon Block ( Core Block )

### Screencast
Basic demo of adding block to editor

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: includes/blocks/icon/block.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Block Information & Attributes File.
+ *
+ * @since 2.0.0
```
Comment (AbhijitNage123) [inline includes/blocks/icon/block.php:5]: x.x.x

Comment (adi3890) [reply includes/blocks/icon/block.php]: updated

**Review by AbhijitNage123 — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: src/blocks/icon/save.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	const blockProps = useBlockProps.save( {
+		className: classnames( `uagb-block-${ block_id }` )
+	} );
+
+	return (
+		<div { ...blockProps }>
+			<div className='uagb-icon-wrapper'>
+				{ iconHtml }
```
Comment (RajkiranBagal) [inline src/blocks/icon/save.js:47]: Is this wrapper div really needed? OR Can we skip it?

Comment (adi3890) [reply src/blocks/icon/save.js:47]: That is required to prevent rotation to wrapper

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**